libs/community/tests/integration_tests/retrievers/test_azure_cognitive_search.py

```diff
@@ -7,22 +7,31 @@
 
 
 def test_azure_cognitive_search_get_relevant_documents() -> None:
-    """Test valid call to Azure Cognitive Search."""
+    """Test valid call to Azure Cognitive Search.
+
```

Comment:
NIT: we renanmed to Azure AI Search now, perhaps change all instances to "azure search"

Reply:
Yes! Planning to open a separate PR with all the changes for this. I saw some docs need to be updated too.